sphinx/prolog.rst

```diff
@@ -0,0 +1,524 @@
+.. |BokehDirective| replace:: :py:class:`~bokeh.sphinxext.bokeh_directive.BokehDirective`
+.. |BokehEnumDirective| replace:: :py:class:`~bokeh.sphinxext.bokeh_enum.BokehEnumDirective`
+.. |BokehGalleryDirective| replace:: :py:class:`~bokeh.sphinxext.bokeh_gallery.BokehGalleryDirective`
+.. |BokehJinjaDirective| replace:: :py:class:`~bokeh.sphinxext.bokeh_jinja.BokehJinjaDirective`
+.. |BokehModelDirective| replace:: :py:class:`~bokeh.sphinxext.bokeh_model.BokehModelDirective`
+.. |BokehOptionsDirective| replace:: :py:class:`~bokeh.sphinxext.bokeh_options.BokehOptionsDirective`
+.. |BokehPaletteGroupDirective| replace:: :py:class:`~bokeh.sphinxext.bokeh_palette_group.BokehPaletteGroupDirective`
+.. |bokeh_palette_group| replace:: :py:class:`~bokeh.sphinxext.bokeh_palette_group.bokeh_palette_group`
+.. |BokehPlotDirective| replace:: :py:class:`~bokeh.sphinxext.bokeh_plot.BokehPlotDirective`
+.. |PlotScriptError| replace:: :py:class:`~bokeh.sphinxext.bokeh_plot.PlotScriptError`
+.. |PlotScriptParser| replace:: :py:class:`~bokeh.sphinxext.bokeh_plot.PlotScriptParser`
+.. |BokehPropDirective| replace:: :py:class:`~bokeh.sphinxext.bokeh_prop.BokehPropDirective`
+.. |BokehReleases| replace:: :py:class:`~bokeh.sphinxext.bokeh_releases.BokehReleases`
+.. |CollapsibleCodeBlock| replace:: :py:class:`~bokeh.sphinxext.collapsible_code_block.CollapsibleCodeBlock`
+.. |collapsible_code_block| replace:: :py:class:`~bokeh.sphinxext.collapsible_code_block.collapsible_code_block`
+.. |ExampleHandler| replace:: :py:class:`~bokeh.sphinxext.example_handler.ExampleHandler`
+.. |Theme| replace:: :py:class:`~bokeh.themes.theme.Theme`
+.. |DummyWebBrowser| replace:: :py:class:`~bokeh.util.browser.DummyWebBrowser`
+.. |EventCallbackManager| replace:: :py:class:`~bokeh.util.callback_manager.EventCallbackManager`
+.. |PropertyCallbackManager| replace:: :py:class:`~bokeh.util.callback_manager.PropertyCallbackManager`
+.. |AttrDict| replace:: :py:class:`~bokeh.util.compiler.AttrDict`
+.. |CoffeeScript| replace:: :py:class:`~bokeh.util.compiler.CoffeeScript`
+.. |CompilationError| replace:: :py:class:`~bokeh.util.compiler.CompilationError`
+.. |CustomModel| replace:: :py:class:`~bokeh.util.compiler.CustomModel`
+.. |FromFile| replace:: :py:class:`~bokeh.util.compiler.FromFile`
+.. |Implementation| replace:: :py:class:`~bokeh.util.compiler.Implementation`
+.. |Inline| replace:: :py:class:`~bokeh.util.compiler.Inline`
+.. |JavaScript| replace:: :py:class:`~bokeh.util.compiler.JavaScript`
+.. |Less| replace:: :py:class:`~bokeh.util.compiler.Less`
+.. |TypeScript| replace:: :py:class:`~bokeh.util.compiler.TypeScript`
+.. |MultiValuedDict| replace:: :py:class:`~bokeh.util.datatypes.MultiValuedDict`
+.. |Options| replace:: :py:class:`~bokeh.util.options.Options`
+.. |BokehDeprecationWarning| replace:: :py:class:`~bokeh.util.warnings.BokehDeprecationWarning`
+.. |BokehUserWarning| replace:: :py:class:`~bokeh.util.warnings.BokehUserWarning`
```

Comment:
It's sorted by fully qualified class name including the package, but it might make sense to sort just by class name. Either way, I'll put some blank lines to make it easier to navigate.